frontend/packages/kubevirt-plugin/integration-tests-cypress/plugins/index.js

```diff
@@ -0,0 +1,63 @@
+// This function is called when a project is opened or re-opened (e.g. due to
+// the project's config changing)
+const wp = require('@cypress/webpack-preprocessor');
+const fs = require('fs');
+
+module.exports = (on, config) => {
```

Comment:
@dtaylor113 I assume all plugins are doing this. Is there a way we can keep the config in place instead of copying it across plugins?